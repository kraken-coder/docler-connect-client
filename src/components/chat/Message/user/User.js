import React from 'react'
import PropTypes from 'prop-types'

const User = ({ user }) => (
  <div className="msg-top">
    <p className="msg-user">{user}</p>
  </div>
)

User.propTypes = {
  user: PropTypes.string.isRequired,
}
export default User
