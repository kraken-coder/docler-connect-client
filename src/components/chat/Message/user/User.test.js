import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import User from './User'

test('Renders Without Crashing', () => {
  const Wrapper = shallow(<User user="username" />)
  expect(toJSON(Wrapper)).toMatchSnapshot()
})
