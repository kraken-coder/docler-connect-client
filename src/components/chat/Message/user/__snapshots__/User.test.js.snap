// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Without Crashing 1`] = `
<div
  className="msg-top"
>
  <p
    className="msg-user"
  >
    username
  </p>
</div>
`;
